src/CoreBundle/Resources/public/javascript/cms.js

```diff
@@ -517,3 +517,47 @@ CHAMELEON.CORE.handleFormAndLinkTargetsInModals = function () {
         $(this).attr('href', link);
     });
 };
+
+CHAMELEON.CORE.initializeEntryAutocomplete = function($element) {
+    $element.typeahead({
+        source: function (query, process) {
+            return $.get($element.data('source-url'), {'term': query}, function (data) {
+                return process(data);
+            });
+        },
+        afterSelect: function(item) {
+            if (typeof item.id !== "undefined") {
+                if ("" !== $element.data('record-url')) {
+                    top.document.location.href = $element.data('record-url') + '&id=' + item.id;
+                }
+            }
+        },
+        autoSelect: false,
+        minLength: 0,
+        showHintOnFocus: true,
+        items: $(window).height() > 600 ? 16 : 8,
+        menu: '<ul class="typeahead dropdown-menu dropdown-menu-right" role="listbox"></ul>X',
```

Comment:
There is an X behind the UL? Is this needed?

Reply:
I would set the dropdown to the left.

Reply:
X removed

left: Normally the input field is on the right of the page. And positioning the dropdown right means that it will open to the left - otherwise it will often be outside of the screen on the right.

Reply:
ok.